line_detection/launch/stereo_gabor.launch

```diff
@@ -0,0 +1,27 @@
+<launch>
+
+  <node pkg="line_detection" name="gabor_linedetection" type="gabor-linedetection.py" ns="left" output="screen">
+    
+    <param name="use_mono" type="bool" value="true" />
+    <param name="use_compressed_format" type="bool" value="false" />
+    <param name="subscriber_image_topic" type="str" value="stereo_camera/left/image_raw" />
```

Comment:
this should be divided into a camera namespace parameter as well, incase we case the camera we are using (e.g. bumblebee or duo)